extensions/wikia/WallNotifications/WallNotifications.class.php

```diff
@@ -387,50 +387,65 @@ protected function sendEmails( array $watchers, WallNotificationEntity $notifica
 			$watcher = $this->getUser( $val );
 			$mode = $watcher->getOption( 'enotifwallthread' );
 
-			if ( !empty( $mode ) && $watcher->getId() != 0 && (
+			if ( !( $watcher->getBoolOption('unsubscribed') === true ) &&
```

Comment:
We can't drop it until all emails are migrated to new extension: https://github.com/Wikia/app/pull/7167#discussion_r30186204